client/lib/load-script/dom-operations.js

```diff
@@ -22,6 +22,14 @@ export function createScriptElement( url ) {
 	return script;
 }
 
+export function hasScriptElement( url ) {
+	const loadedScripts = document.getElementsByTagName( 'script' );
+
+	const scriptMatchingUrl = Array.from( loadedScripts ).find( scriptTag => scriptTag.src === url );
```

Comment:
Array.prototype.find isn't supported on IE11. We don't currently polyfill the array prototype, so this probably breaks.